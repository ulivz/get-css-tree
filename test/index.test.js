import getCSSTree from '../'
import fs from 'fs'
import path from 'path'

test('should get correct css tree', () => {
  const htmlstring = fs.readFileSync(path.resolve(__dirname, '__fixtures__/test.html'), 'utf-8')
  const csstree = getCSSTree(htmlstring)
  expect(csstree).toMatchSnapshot()
})
