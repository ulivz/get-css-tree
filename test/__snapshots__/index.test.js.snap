// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get correct css tree 1`] = `
".header {
	.tabs {
	}
}
.content {
	.sidebar {
	}
}
.footer {
}"
`;
